examples/automatic_labelling/main.tf

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "archive" {
+  version = "~> 1.1"
+}
+
+provider "google" {
+  version = "~> 1.20"
+}
+
+provider "random" {
+  version = "~> 2.0"
+}
+
+resource "random_pet" "main" {
+  separator = "-"
+}
+
+module "event_function" {
```

Comment:
Since this is a simple example, can we strip down some of these variables to rely on the default? We can add other examples that fully exercise all of the variables provided but I think there's value in having a really bare bones version for people to get started with.

Reply:
That's fine by me.